components/widgets/iFrame.js

```diff
@@ -0,0 +1,85 @@
+import iFrame from 'react-iframe';
+
+var __assign =
```

Comment:
Are we actually using the iFrame anywhere? If not, it may make sense to remove this file and the package.json entry

Reply:
True. I'll remove that from my next PR. I was originally pulling in the iFrame to embed a google sheet so we could have a table of raw data. But now I found materials-ui, seems like we can use that instead. So I'll remove iFrame reference.